etc/helm/pachyderm/templates/proxy/service.yaml

```diff
@@ -0,0 +1,83 @@
+{{- if .Values.proxy.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: pachyderm-proxy
+    suite: pachyderm
+    {{- if .Values.proxy.service.labels -}}
+    {{- toYaml .Values.proxy.service.labels | nindent 4 }}
+    {{- end }}
+  {{- if .Values.proxy.service.annotations }}
+  annotations:
+    {{- toYaml .Values.proxy.service.annotations | nindent 4 }}
+  {{- end }}
+  name: pachyderm-proxy
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: {{ .Values.proxy.service.type }}
+  ports:
+    - name: http-port
```

Comment:
left another comment, but are we planning on exposing https port at some point?

Reply:
Yup, CORE-577.